sed/config/NXmpes_config-HEXTOF.json

```diff
@@ -159,25 +150,27 @@
     "frequency/@units": "kHz",
     "associated_beam": "/entry/instrument/beam_pump"
   },
-  "/ENTRY[entry]/INSTRUMENT[instrument]/beam_TYPE[beam_pump]": {
+  "/ENTRY[entry]/INSTRUMENT[instrument]/beamTYPE[beam_pump]": {
     "distance": 0.0,
     "distance/@units": "mm",
     "incident_wavelength": "@attrs:metadata/scientificMetadata/Laser/wavelength/value",
     "incident_wavelength/@units": "@attrs:metadata/scientificMetadata/Laser/wavelength/unit",
-    "pulse_duration": "@attrs:metadata/scientificMetadata/Laser/wavelength/pulse duration/value",
-    "pulse_duration/@units": "@attrs:metadata/scientificMetadata/Laser/wavelength/pulse duration/unit",
-    "incident_polarization": "@attrs:metadata/instrument/beam/pump/incident_polarization",
+    "incident_energy": "@attrs:metadata/scientificMetadata/Laser/energy/value",
+    "incident_energy/@units": "@attrs:metadata/scientificMetadata/Laser/energy/unit",
```

Comment:
Calculate from wavelength, or add to SciCat

Reply:
So this is photon energy, 1.2 eV for 1030 nm, not a pulse energy?

Reply:
see https://fairmat-nfdi.github.io/nexus_definitions/classes/base_classes/NXbeam.html#nxbeam
Yes, photon energy.
There is also pulse_energy

Reply:
I put conversion into the notebook. If this layout is final, can also be added in the Flash metadata code.